pkgs/applications/networking/instant-messengers/nheko/default.nix

```
@@ -59,12 +73,18 @@ mkDerivation rec {
     qtquickcontrols2
     qtgraphicaleffects
     qtkeychain
-  ] ++ lib.optional stdenv.isDarwin qtmacextras;
+  ] ++ lib.optional stdenv.isDarwin qtmacextras
+    ++ lib.optionals voipSupport voipDeps;
```

```suggestion
    ++ lib.optionals voipSupport (with gst_all_1; [
      gstreamer
      gst-plugins-base
      gst-plugins-good
      gst-plugins-bad
      libnice
  ]);
```

I think this is even cleaner because we do not introduce a variable and let in block.

---

All right...